pages/faq.js

```diff
@@ -1,92 +1,123 @@
-          This is used for determining which packages should receive the revenue generated.
-          Additionally, the version of the package manager used is recorded for debugging purposes.
-          Our package manager wrapper is open source, so feel free to <TextLink text='inspect the code on GitHub.' external href='https://github.com/flossbank/cli' />
-        </Text>
-        <SubHeading>Which operating systems are supported?</SubHeading>
-        <Text marginBottom='1.5rem'>
-          Windows, Mac, and Linux.
-        </Text>
-        <SubHeading>How can I uninstall Flossbank?</SubHeading>
-        <Text marginBottom='1.5rem'>
-          At any point, you can run the following command to uninstall:
-        </Text>
-        <Box position='relative' marginBottom='1.5rem'>
-          <Code
-            padding='.75rem'
-            backgroundColor='lightRock'
-            color='boulder'
-            width='100%'
-          >
-            flossbank uninstall
-          </Code>
-          <FBButton
-            // hide on mobile
-            display={{ base: 'none', md: 'flex' }}
-            alignItems='center'
-            onClick={onCopy}
-            padding='.5rem'
-            fontSize='.85rem'
-            minW='5rem'
-            backgroundColor='puddle'
-            color='ocean'
-            position='absolute'
-            right='0'
-            top='.25rem'
-            borderRadius='0'
-            aria-live='assertive'
-            _hover={{
-              backgroundColor: 'ocean',
-              color: 'white'
-            }}
-          >
-            {hasCopied ? 'Copied' : 'Copy'}{' '}
-            <span className='sr-only'>command to uninstall</span>
-            <Icon name='copy' size='1rem' marginLeft='.5rem' />
-          </FBButton>
-        </Box>
-        <SubHeading>I still have a question</SubHeading>
-        <Text marginBottom='1.5rem'>
-          <TextLink text='Contact us' href='contact' /> and we'll address it ASAP.
-        </Text>
-      </Card>
+    <PageWrapper title='FAQs'>
+      <Section
+        display='flex'
+        justifyItems='center'
+        flexDirection='column'
+        alignItems='center'
+        padding={{ base: '3rem 1.5rem', lg: '3rem 7.5rem 6rem' }}
+        backgroundColor='lightRock'
+      >
+        <UnderlinedHeading as='h1' text='FAQs' align='center' />
+        <Card shadowSz='lg' maxW='60rem'>
+          <FAQHeading>What do the ads look like?</FAQHeading>
+          <Box marginBottom='1.5rem'>
+            <Image
+              src='/images/flossbank_ads_demo.gif'
+              margin='1.5rem auto 0'
+            />
+          </Box>
+          <Divider />
+          <FAQHeading>What package managers do you wrap?</FAQHeading>
+          <Text marginBottom='1.5rem'>
+            We are currently wrapping{' '}
+            <TextLink href='https://www.npmjs.com/' external text='npm' /> and{' '}
+            <TextLink href='https://yarnpkg.com/' external text='Yarn' />. We
```

Comment:
Reason we need the target=blank is we can’t link out to yarn and npm if it redirects the user from the current tab